scripts/ci/check_pr_merge_readiness.py

```
@@ -0,0 +1,271 @@
+from __future__ import annotations
+
+import argparse
+import http.client
+import json
+import os
+import re
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)